flake.nix

```diff
@@ -35,5 +35,8 @@
       modules = import ./pkgs/modules {
         inherit pkgs;
       };
+      upgrade-maps = import ./pkgs/upgrade-maps {
+        inherit pkgs;
+      };
```

Comment:
It's being used in `check_upgrade_maps.py` for verification. Is that a valid use case? Seem like having to build a file is a bit heavy for that.

Reply:
You can make a new "checks" flake output that runs your python check program and pass in the upgrade map that way. This will also make it run when you do `nix flake check`.

Reply:
:+1: